extensions/amp-story/0.1/amp-story-auto-ads.js

```diff
@@ -226,8 +246,58 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     const attributes = /** @type {!JsonObject} */ (Object.assign({},
         defaultAttrs, configAttrs));
 
-    return createElementWithAttributes(
+    const adElement = createElementWithAttributes(
```

Comment:
`this.currentAdElement_ =` to save one line.

Reply:
done